### DISCUSSION TITLE
How to proceed

### DISCUSSION THREAD

**Discussion by smhg** [smhg/gettext-parser#100] [General]
@erikyo I continue [the discussion](https://github.com/smhg/gettext-parser/issues/82#issuecomment-2109940766) here.

I think we basically are stuck with a Git Flow vs GitHub Flow approach.

I have little time, spread out in bits and chuncks. I can't read X issues, internal PR's, comments spread out over repositories. I also can't compare `master` to the `development` branch as it is a mix of many different changes which requires a lot of time for me to filter and figure out all their intricacies.
In order for me to release your work, I need small chuncks in the form or PRs on master. Those run the tests and their atomicity means I can squash and merge some of them quickly. Others might not even require me looking at them (the ones I don't understand anyway). But for those I need to see _what_ they contain, not so much _how_ they work. Actually, if they are atomic, I don't have to be involved much at all.

There might be other ways, this is _a_ way how I imagine things can proceed.

Edit: I want to release things that can be released. Not so much to ship them, but to unclutter what remains.

**Comment by erikyo** [comment]
hi @smhg,

Thank you for your feedback. I'd like to clarify a few points and discuss how we can proceed effectively.

1. **Lack of Clarity**: Your response in the thread [here](https://github.com/smhg/gettext-parser/issues/82#issuecomment-2092514379) gave me the impression of a certain disinterest in managing the project, so I acted accordingly. If you have now changed your mind, I’d like to understand better how we can move forward without redoing all the work. If we can't reach an agreement, I'm open to forking the project.
2. **Maintainer's Responsibility**: I understand that time is limited, but as the maintainer, it's essential to at least read the PRs and related comments. Without this minimum level of review, it becomes difficult to maintain a high quality in the project. If things are accepted without review, the stability and coherence of the code might be compromised.
3. **GitHub Tools**: GitHub provides advanced tools for comparing branches and commits. I’m not sure what specific difficulties you're encountering. https://docs.github.com/en/pull-requests/committing-changes-to-your-project/viewing-and-comparing-commits/comparing-commits

I understand that you prefer smaller, atomic PRs to master, and I can adapt to this method in the future. However, reorganizing the work already done requires significant time and effort. Can we find a compromise that allows us to integrate the existing work without too many hurdles?

Thank you for your understanding, and I look forward to your response.

**Reply by smhg** [reply]
Seeing how hard it will be for me the next 2 months to invest time in this, it seems wise you continue your fork for now.
I'm really only able to review, merge and release small, concise PR's currently. The reality of quite a lot open source libraries I guess.

I definitely plan to take up some of the easy parts as soon as possible. After that, I hope to pick things up where we left them.

Thank you!
